superset-frontend/src/SqlLab/components/ResultSet/index.tsx

```diff
@@ -343,7 +343,7 @@ export default class ResultSet extends React.PureComponent<
           metrics: [],
           groupby: [],
           time_range: 'No filter',
-          viz_type: 'table',
+          viz_type: '',
```

Comment:
In bootstrap data we have `common.conf` which contains `DEFAULT_VIZ_TYPE` (you can check the page source and look for `data-bootstrap="..."`). Instead of passing this as an empty string, I would propose one of the following:
1. passing the `DEFAULT_VIZ_TYPE` to this component from wherever the bootstrap data is available, or
2. Removing it all together, and then just replacing it with `DEFAULT_VIZ_TYPE` is if's undefined in the backend.

Reply:
Looks like you went with door #2. Resolving this comment :)